Project.toml

```diff
@@ -14,12 +14,13 @@ StyledStringsExt = "StyledStrings"
 
 [compat]
 FixedPointNumbers = "0.5, 0.6, 0.7, 0.8"
-StyledStrings = "1"
+# StyledStrings = "1" # FIXME: workaround for unregistered StyledStrings
```

Comment:
This should be fixed in the close future, but currently `StyledStrings` is unregistered and it should be v1 for the time being.
We are touching the internals of `StyledStrings`, so we run the risk of incompatibility. And that can't be prevented by the `"1"` cap anyway.
（In other words, *if we follow the principle*, we need to specify the patch version here.）

Reply:
What constitutes a breaking change is ambiguous. If `SimpleColor` added a feature to have an alpha component, it would not break the code, but it would go against the user's intuition.
That is just an example. `StyledStrings` is community owned, although you are the lead developer. I don't believe anything bad will happen, but there is no guarantee how it will turn out.
*For example, CI could cease to function effectively even though no one has done anything wrong.*